### PR TITLE
feat(ngx-inform): Setup ngx-inform package

### PR DESCRIPTION
**Pull request by IbenTesara**
Setup for the `ngx-inform` package, which will be the home for tooltips, toasts and snackbars.

### CODE REVIEW

File: libs/inform/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	"name": "@studiohyperdrive/ngx-inform",
+	"version": "18.0.0",
+  "description": "A lightweight ARIA compliant customizable approach for common and complex inform flows in Angular.",
+  "keywords": [
+    "angular",
+    "angular2",
+    "inform",
+    "ARIA",
+    "tooltip",
+    "tip"
```
Comment (WHeirstrate) [inline libs/inform/package.json:11]: Accessibility

File: tsconfig.base.json [hunk 2]
```diff
@@ -22,13 +22,14 @@
 			"@ngx/layout": ["libs/layout/src/public-api.ts"],
 			"@ngx/store": ["libs/store/src/public-api.ts"],
 			"@ngx/table": ["libs/table/src/public-api.ts"],
+			"@ngx/tour": ["libs/tour/src/index.ts"],
 			"@ngx/utils": ["libs/utils/src/public-api.ts"],
-			"@ngx/tour": ["libs/tour/src/index.ts"]
```
Comment (DenisValcke) [inline tsconfig.base.json:26]: Why does this one not match the others?

**Review by WHeirstrate — APPROVED** [review]

File: tsconfig.base.json (hunk 2 above)
Comment (IbenTesara) [reply tsconfig.base.json:26]: That's the new approach of Nx, it has been like that for quite a while now (was already the case with the ngx-tour package). I don't know if there's a migration script for it, but I'm hesitant to edit this if this is their new approach 😅

Comment (DenisValcke) [reply tsconfig.base.json:26]: Fair enough, maybe we should create a task on the board to do the same for the existing packages.

**Review by DenisValcke — APPROVED** [review]

**Pull request merged**